SPECS/kmod-drbd.spec

```diff
@@ -1,7 +1,7 @@
 Name: kmod-drbd
 Summary: Kernel driver for DRBD
-Version: 9.2.10
-Release: 1.5%{?dist}
+Version: 9.2.11
```

Comment:
If it's from a branch, are we sure this is exactly version 9.2.11?

Reply:
It's the project makefile that generate the tarball and define the version

Reply:
But is it the real version when it's coming from a branch? Does there exist a 9.2.11 drbd whose sources differ from that branch you used? If yes, there are packager best practices to represent the fact that it's a developer or parallel release.

Reply:
It's got some more patches I guess, how would we represent this fact?

Reply:
That's where the fun is: it depends on how exactly this release differs from official releases. Whether it's a pre-release for 9.2.11, or a pre-release for 9.2.12, or a 9.2.11 with temporary patches...

Reply:
Let's discuss it this afternoon?